apps/omg_watcher/lib/exit_processor.ex

```diff
@@ -385,6 +382,34 @@ defmodule OMG.Watcher.ExitProcessor do
     end
   end
 
+  defmacro is_deposit(blknum) do
+    {:ok, interval} = OMG.Eth.RootChain.get_child_block_interval()
+
+    quote do
+      rem(unquote(blknum), unquote(interval)) != 0
+    end
+  end
+
+  defp get_standard_exit_id(
```

Comment:
put all the machinery (this and `is_deposit`) to calculate the tx hash into `Core` and under unit tests pls. For non-deposit txs it must circle back into here, so it might require a bit more figuring out, but this functionality should be covered by tests.